db/migrations/admin-migrations/2024-07-15.0-migrate-data-v1-to-v2-function.sql

```diff
@@ -0,0 +1,99 @@
+-- +migrate Up
+
+-- +migrate StatementBegin
+CREATE OR REPLACE FUNCTION import_tenant_data_from_v1_to_v2(tenant_name TEXT) RETURNS void AS $$
```

Comment:
WDYT about renaming it from `import...` to `migrate...`?

Reply:
Addressed in b405615acc94fe7a87822fa65